### ISSUE TITLE
Static html export

### ISSUE THREAD

**Issue by sinedied** [sinedied/hads#2]
Add an option to generate static html files from the documentation, like `--export <out_folder>`

**Comment by japf** [comment]
That's a must have for the scenario I'm thinking about :-)

**Comment by dalgard** [comment]
I haven't been able to find a good zero-config tool for displaying the simple Markdown files in my repo/package that can also compile them into a static HTML site.

In other words, this feature would close a big hole in the tooling landscape for me 🙂

Also – thanks so much for creating this, I absolutely love the idea and the beautiful implementation ✨🙏

**Comment by rhucle** [comment]
I'd like to help with this. @sinedied Do you have any thoughts on how you'd like it done?

**Comment by sinedied** [comment]
Sorry for the delay!

If you want to get started, a good approach I think would be:
- use the list of all doc files available generated for the search index, to go through each one and render it to a file
- while rendering to a file, 2 things have to be taken care of:
  * disable/hide search bar, as it won't work as it is for static files
  * change way markdown generates links to other local files, to add .html suffix to the links

Basically, that should do the trick :)

**Comment by docset** [comment]
Great. I'll take a look this weekend.

**Comment by rhucle** [comment]
Great. I'll take a look this weekend.

**Comment by rhucle** [comment]
https://github.com/sinedied/hads/pull/45
